packages/app-tests/e2e/document-flow/settings-step.spec.ts

```diff
@@ -1,16 +1,148 @@
 import { expect, test } from '@playwright/test';
+import dotenv from 'dotenv';
+import path from 'path';
 
 import {
   seedBlankDocument,
   seedDraftDocument,
   seedPendingDocument,
 } from '@documenso/prisma/seed/documents';
+import { seedUserSubscription } from '@documenso/prisma/seed/subscriptions';
+import { seedTeam, unseedTeam } from '@documenso/prisma/seed/teams';
 import { seedUser, unseedUser } from '@documenso/prisma/seed/users';
 
 import { apiSignin } from '../fixtures/authentication';
 
+dotenv.config({ path: path.resolve(__dirname, '../../../../', '.env.local') });
```

Comment:
Should be fine to load the ENV into playwright right?

Reply:
Yeah it should be

Reply:
Although it should already have the env for the most part 🤔